packages/client/components/TransactionListItem.tsx

```diff
@@ -38,37 +42,51 @@ export function TransactionListItem({ transaction, walletAddress }: Props) {
     }
 
     return (
-        <Box style={styles.transactionItemWrapper}>
-            <HStack>
-                <FontAwesomeIcon
-                    icon={isIncommingTransaction ? faCircleArrowDownLeftCustom : faCircleArrowUpRightCustom}
-                    style={isIncommingTransaction ? styles.receiveIcon : styles.sendIcon}
-                    size={30}
-                />
-                <VStack style={styles.verticalStack}>
-                    <HStack>
-                        <Text style={styles.transactionsAddress}>
-                            {formatTransactionAddress(transaction.transactionAddress)}
-                        </Text>
-                        <Text
-                            color={isIncommingTransaction ? "success.600" : "text.600"}
-                            style={isIncommingTransaction ? styles.transactionAmountIn : styles.transactionAmountOut}
-                        >
-                            {isIncommingTransaction ? "+" : "-"}
-                            {formatAmount(transaction.amount)}
-                        </Text>
-                    </HStack>
-                    <HStack>
-                        <Text color="text.500" style={styles.transactionDate}>
-                            {getFromattedDate(transaction.createdAt)}
-                        </Text>
-                        <Text color="text.500" style={styles.transactionCurrency}>
-                            {getCurrencyType()}
-                        </Text>
-                    </HStack>
-                </VStack>
-            </HStack>
-        </Box>
+        <Pressable
+            onPress={() =>
+                navigation.navigate("TransactionDetailsScreen", {
+                    title: "Monday",
+                    transaction: transaction,
+                    walletAddress: walletAddress,
+                })
+            }
+        >
+            <Box style={styles.transactionItemWrapper}>
+                <HStack>
+                    <FontAwesomeIcon
+                        icon={isIncommingTransaction ? faCircleArrowDownLeftCustom : faCircleArrowUpRightCustom}
+                        style={isIncommingTransaction ? styles.receiveIcon : styles.sendIcon}
+                        size={30}
+                    />
+                    <VStack style={styles.verticalStack}>
+                        <HStack>
+                            <Text style={styles.transactionsAddress}>
+                                {formatTransactionAddress(transaction.transactionAddress)}
+                            </Text>
+                            <Text
+                                color={isIncommingTransaction ? "success.600" : "text.600"}
```

Comment:
Replace `text.600` with `text.700`
```suggestion
                                color={isIncommingTransaction ? "success.600" : "text.700"}
```